src/components/ButtonDropdown.vue

```diff
@@ -10,7 +10,7 @@
       </span>
     </button>
     <div v-show="isOpen" class="button-dropdown-content" :style="dropdownStyle">
-      <slot></slot>
+      <slot v-if="isOpen"></slot>
```

Comment:
I use `v-if` to intentionally cause a destroy and re-mounting of the child component wrapped by the `slot`.  Since we may not want to force that behavior on all cases of ButtonDrowdown, it just occurred to me I can add an optional param to ButtonDropdown to allow us to specify show/hide or hard rerender.

Thoughts?